core/pta/it_notif.c

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2023 Linaro Limited
+ */
+
+#include <kernel/notif.h>
+#include <kernel/pseudo_ta.h>
+#include <pta_it_notif.h>
+#include <tee_api_defines.h>
+#include <tee_api_defines_extensions.h>
+#include <util.h>
+
+static TEE_Result activate_it(uint32_t param_types,
+			      TEE_Param params[TEE_NUM_PARAMS])
+{
+	if (param_types != TEE_PARAM_TYPES(TEE_PARAM_TYPE_VALUE_INPUT,
+					   TEE_PARAM_TYPE_NONE,
+					   TEE_PARAM_TYPE_NONE,
+					   TEE_PARAM_TYPE_NONE))
+		return TEE_ERROR_BAD_PARAMETERS;
+
+	return notif_it_set_state(params[0].value.a, true /* enable */);
```

Comment:
Since we have a single `notif_it_set_state()` why not supply `params[0].value.b` as the second parameter? It should reduce the interface a bit.

Reply:
Ok. And same for wakeup state commands.